dependencies.properties

```diff
@@ -32,5 +32,6 @@ maven.org_threeten_threetenbp=org.threeten:threetenbp:1.3.3
 
 # Testing.
 maven.junit_junit=junit:junit:4.13
+maven.org_hamcrest_hamcrest_core=org.hamcrest:hamcrest-core:1.3
```

Comment:
This dependency is for running JUnit test manually, before JUnit 4.11 it's wrapped along with junit package. Now it has to be explicitly added.

Reply:
Could you please add that note as a comment?

Reply:
Sure, done.

Reply:
Not sure about `gradle` side, but yes if this is only needed from bazel build, I can move this to the `repositoreis.bzl`. Done.

Reply:
moved it back based on offline talk that we may want to refactor the dependencies all together in future.

Reply:
Thanks.